webpack.config.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path');
+
+module.exports = {
+  entry: __dirname + '/src/main.jsx',
+  output: {
+    path: __dirname + '/build/js',
+    filename: '[name].bundle.js'
+  },
+  externals: {
+    'react': 'React',
```

Comment:
Does `-p` mean production? For teach, we use the define plugin to set `NODE_ENV=production`, which might do the same thing--that way e.g. `React.addons.TestUtils` isn't included in the bundle and stuff.